### DISCUSSION TITLE
files created automatically by workflows

### DISCUSSION THREAD

**Discussion by johanneswerner** [nfdi4plants/ARC-specfication#45] [Comments and Suggestions]
A paragraph on how to deal with files automatically created by workflows (e.g. index files, temporary files) might be useful. My suggestion is to not add such files to an ARC (to save disk space), however the workflow should ensure that the file keeps the same (e.g. by checksum comparison). Of course, this is just a small perspective on this topic, please share your opinion on this topic.

**Comment by chgarth** [comment]
Interesting idea. CWL should technically not produce "temporary files". Checksum comparisons may be difficult for some files (e.g. from randomized algorithms).

I think we could teach ARC commander to by default only add files listed as explicit CWL outputs to the repository when adding a run.
